logging/loki/deploy-loki.sh

```
@@ -0,0 +1,289 @@
+    --wait --timeout 5m
+fi
+success "Loki deployed successfully"
+
+# =============================================================================
+# STEP 4: Deploy Grafana
+# =============================================================================
+info "Step 4: Deploying Grafana (chart: $GRAFANA_CHART_VERSION)..."
+
+# Read from environment or prompt securely
+if [ -z "$GRAFANA_PASSWORD" ]; then
+  read -s -p "Enter Grafana admin password: " GRAFANA_PASSWORD
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)